policyengine_us/variables/gov/states/az/tax/income/exemptions/az_blind_exemption.py

```diff
@@ -0,0 +1,38 @@
+from policyengine_us.model_api import *
+
+
+class az_blind_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona blind exemption"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        # First get their filing status.
+        filing_status = tax_unit("filing_status", period)
+
+        # Determine whether spouse is eligible.
+        joint = filing_status == filing_status.possible_values.JOINT
+
+        # Then get the AZ blind exemptions part of the parameter tree.
+        p = parameters(
+            period
+        ).gov.states.nj.tax.income.exemptions.blind_or_disabled
+
+        # Get the individual blind status and disabled.
+        blind_head = tax_unit("blind_head", period)
```

Comment:
For concision (spouse too)
```suggestion
        head_eligible = tax_unit("blind_head", period).astype(int)
```